lib/charms/operator_libs_linux/v2/snap.py

```diff
@@ -409,6 +412,17 @@ def unhold(self) -> None:
         """Remove the refresh hold of a snap."""
         self._snap("refresh", ["--unhold"])
 
+    def alias(self, application: str, alias: str = None) -> None:
+        """Create an alias for a given application.
+
+        Args:
+            application: application to get an alias.
+            alias: (optional) name of the alias
+        """
+        if not alias:
```

Comment:
I prefer the more explicit `if alias is None` in cases like this. Empty string is probably not valid here, but in general it might be, and it's good to distinguish between not provided (None) and set to empty string.

Reply:
Updated.